command/init.go

```diff
@@ -28,10 +32,135 @@ func (c *InitCommand) Run(args []string) int {
 	flags.IntVar(&recoveryThreshold, "recovery-threshold", 3, "")
 	flags.Var(&recoveryPgpKeys, "recovery-pgp-keys", "")
 	flags.BoolVar(&check, "check", false, "")
+	flags.BoolVar(&auto, "auto", false, "")
+	flags.StringVar(&consulService, "consul-service", "", "")
```

Comment:
Can we make the flag name, `consul-name` or `consul-service-name` ?  And can we specify a default, like `vault` since that's what's used elsewhere?  There's an unexported constant that we should export and just reuse for the default value: https://github.com/hashicorp/vault/blob/master/physical/consul.go#L43